Configurations/unix-Makefile.tmpl

```diff
@@ -1260,7 +1260,8 @@ providers/fips.module.sources.new: configdata.pm
 		   crypto/ec/asm/*.pl \
 		   crypto/modes/asm/*.pl \
 		   crypto/sha/asm/*.pl \
-		   crypto/x86_64cpuid.pl; do \
+		   crypto/*cpuid.pl crypto/*cpuid.S \
+		   crypto/*cap.c; do \
```

Comment:
Nit: I suggest the files be on their own line, but there is prior precedence for having two files per line.